modules/Bio/EnsEMBL/Mapper.pm

```diff
@@ -358,15 +374,16 @@ sub map_coordinates {
               $target_start, $target_end, $pair->{'ori'}*$strand, $cs );
       #and finally, the IndelCoordinate object with
       $res = Bio::EnsEMBL::Mapper::IndelCoordinate->new( $gap, $coord );
+      
     } else {
-      # start is somewhere inside the region
-      if ( $pair->{'ori'} == 1 ) {
-        $target_start = $target_coord->{'start'} + ( $start - $self_coord->{'start'} );
-      } else {
-        $target_end =
-          $target_coord->{'end'} - ( $start - $self_coord->{'start'} );
-      }
+        # start is somewhere inside the region
+        if ( $pair->{'ori'} == 1 ) {
+          $target_start = $target_coord->{'start'} + ( $start - $self_coord->{'start'} );
+        } else {
+          $target_end = $target_coord->{'end'} - ( $start - $self_coord->{'start'} );
+        }
 
```

Comment:
Indeed, it all looks reasonable, and being able to truncate those trailing gaps seems useful. But I agree, would you mind just going through it again and look at your spacing, both indentation and consistent inter-variable spacing? (ie. $var1,$var2) Thanks, sorry for being picky.